library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -0,0 +1,102 @@
+package org.xmtp.android.library
+
+import java.util.Date
+
+sealed class ConversationContainer {
```

Comment:
You might not need this. The container thing was my first attempt at allowing exporting/importing of conversations, but it's a different format than the JS uses. I eventually added the ability to use the JS format for importing in https://github.com/xmtp/xmtp-ios/pull/53, and it might make sense to just use that for import/export in Kotlin instead of the container thing.

Reply:
Nice catch I wasn't even using this code anyways so removed it!